### DISCUSSION TITLE
Security Onion Essentials training available now for FREE!

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#1734] [Announcements]
Security Onion Essentials training is now available on our new website!
https://securityonion.net/training

![Screen Shot 2020-10-29 at 1 13 57 PM](https://user-images.githubusercontent.com/1659467/97608387-a9fd7800-19e8-11eb-8b9d-e6b237fd1b81.png)

**Comment by yerchosi** [comment]
Are the slides used in 'Security Onion Essentials' publicly available?

**Reply by dougburks** [reply]
If you missed something in the video, you can certainly watch the video again. :smile: 

We are not publishing the slides separately as the material is under copyright.
